script.js

```diff
@@ -50,4 +50,393 @@ menuIcon.addEventListener('keydown', (e) => {
+
+  popupImg.append(
+    popupMobileImage,
+    popupDesktopImage,
+  );
+
+  const containerForHeaderContainerAndPopupButtons = document.createElement('div');
+  containerForHeaderContainerAndPopupButtons.classList.add('container_for_header_container_and_popup_buttons');
+
+  const headerContainer = document.createElement('div');
+  headerContainer.classList.add('header_container');
+
+  const popupHeader = document.createElement('h3');
+  popupHeader.classList.add('popup_header');
+  popupHeader.innerHTML = projectInfo[i].popupCardHeader;
+
+  headerContainer.appendChild(
+    popupHeader,
+  );
+
+  const popupButtonsDesktop = document.createElement('div');
+  popupButtonsDesktop.classList.add('popup_buttons_desktop');
+
+  const seeSourceBtn = document.createElement('button');
+  seeSourceBtn.type = 'button';
+  seeSourceBtn.classList.add('see-project');
+  seeSourceBtn.innerHTML = `
+        See Source
+          <span><img src="assets/popup/icons/see-source-icon.svg"></span>
+    `;
+
+  const seeLiveBtn = document.createElement('button');
+  seeLiveBtn.type = 'button';
+  seeLiveBtn.classList.add('see-project');
+  seeLiveBtn.innerHTML = `
+        See Live
+          <span><img src="assets/popup/icons/see-live-icon.svg"></span>
+    `;
+
+  const seeLiveBtnDesktop = document.createElement('button');
+  seeLiveBtnDesktop.type = 'button';
+  seeLiveBtnDesktop.classList.add('see-project');
+  seeLiveBtnDesktop.innerHTML = `
+        See Live
+          <span><img src="assets/popup/icons/see-live-icon.svg"></span>
+    `;
+
+  const seeSourceBtnDesktop = document.createElement('button');
+  seeSourceBtnDesktop.type = 'button';
+  seeSourceBtnDesktop.classList.add('see-project');
+  seeSourceBtnDesktop.innerHTML = `
+        See Source
+          <span><img src="assets/popup/icons/see-source-icon.svg"></span>
+    `;
+
+  popupButtonsDesktop.append(
+    seeLiveBtnDesktop,
+    seeSourceBtnDesktop,
+  );
+
+  containerForHeaderContainerAndPopupButtons.append(
+    headerContainer,
+    popupButtonsDesktop,
+  );
+
+  const mobileLanguage = document.createElement('div');
+  mobileLanguage.classList.add('mobile_language_section');
+
+  const languageMobileContainer = document.createElement('ul');
+  languageMobileContainer.classList.add('language-section');
+
+  const popupLanguageMobileItem1 = document.createElement('li');
+  popupLanguageMobileItem1.innerHTML = [projectInfo[i].popupLanguageMobile[0]];
+  popupLanguageMobileItem1.classList.add('pro-lang');
+
+  // popup mobile language item2
+  const popupLanguageMobileItem2 = document.createElement('li');
+  popupLanguageMobileItem2.innerHTML = [projectInfo[i].popupLanguageMobile[1]];
+  popupLanguageMobileItem2.classList.add('pro-lang');
+
+  // popup mobile language item3
+  const popupLanguageMobileItem3 = document.createElement('li');
+  popupLanguageMobileItem3.innerHTML = [projectInfo[i].popupLanguageMobile[2]];
+  popupLanguageMobileItem3.classList.add('pro-lang');
```

Comment:
- [ ] Kindly check your project languages design. They are not aligned as it is in the Figma design. Kindly fix this.

**This is the desktop design:**

![image](https://user-images.githubusercontent.com/86318284/202445087-8a40af07-cb84-4d53-aa1e-4e774d5fdc5a.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202445219-9c110d20-b095-4292-946c-098bf8ee06a0.png)

**Note** I have realized that the trend is the same for the mobile view as well. Kindly also make this design change to the mobile styling of the popup as well.